src/world/area_arn/arn_11/arn_11.h

```diff
@@ -4,6 +4,6 @@
 
 #define NAMESPACE arn_11
 
-extern Script N(main);
-extern Script N(80240050);
+extern Script N(main); 
+extern Script N(80240050); 
```

Comment:
Remove these trailing spaces pls :shipit:

Reply:
How? This is what I need format.sh for lol